spec/face.html

```diff
@@ -1098,6 +1098,122 @@
         </dl>
       </section>
     </section>
+    <section class='informative'>
+      <h2>
+        Examples
+      </h2>
+      <h3>
+        Start/Stop face module
+      </h3>
+      <pre class='example highlight'>
+        var previewStream;
+        var ft;
+
+        // Start stream firstly, then start face module.
+        startButton.onclick = function(e) {
+          navigator.mediaDevices.getUserMedia(constraints)
+              .then(function(stream) {
+                // Wire the media stream into a &lt;video&gt; element for preview.
+                previewStream = stream;
+                var previewVideo = document.querySelector('#previewVideo');
+                previewVideo.srcObject = stream;
+                previewVideo.play();
+  
+                try {
+                  ft = new realsense.Face.FaceModule(stream);
+                } catch (e) {
+                  console.log('Failed to create face module: ' + e.message);
+                  return;
+                }
+        
+                ft.onready = function(e) {
+                  console.log('Face module ready to start');
+                  // The stream got ready, we can start face module now.
+                  ft.start().then(
+                      function() {
+                        console.log('Face module start succeeds');},
+                      function(e) {
+                        console.log('Face module start failed: ' + e.message);}); 
+                };
+        
+                ft.onprocessedsample = function(e) {
+                  console.log('Got face module processedsample event.');
+                  ft.getProcessedSample(false, false).then(function(processed_sample) {
+                    console.log('Got face module processedsample data.');
+                    // Use processed_sample.faces data to work for you.
```

Comment:
Please give a very basic sample of `processedSample.faces` usage, say print how many faces are detected.